auth/.js/src/main/scala/io/udash/auth/AuthView.scala

```diff
@@ -1,15 +1,27 @@
 package io.udash.auth
 
+import io.udash.auth.PermissionCombinator.AllowAll
 import scalatags.JsDom.all._
 
 trait AuthView {
   /** Renders provided `view` only if user context has required permissions. */
   def require(permission: PermissionCombinator, requireAuthenticated: Boolean = false)(view: => Modifier)(implicit userCtx: UserCtx): Modifier =
-    if ((!requireAuthenticated || userCtx.isAuthenticated) && permission.check(userCtx)) view else ()
+    if (canRenderView(permission, requireAuthenticated)) view else ()
 
   /** Renders provided `view` only if user is authenticated. */
   def requireAuthenticated(view: => Modifier)(implicit userCtx: UserCtx): Modifier =
-    require(PermissionCombinator.AllowAll, requireAuthenticated = true)(view)
+    require(AllowAll, requireAuthenticated = true)(view)
+
+  /** Renders provided view if user is authenticated or fallback view otherwise */
+  def requireWithFallback(permission: PermissionCombinator, requireAuthenticated: Boolean)
+    (view: => Modifier, fallback: => Modifier)(implicit userCtx: UserCtx): Modifier =
```

Comment:
I don't know the most common usage patterns, but with this signature we lock ourselves out of convenient syntax for creating main view, like:
```scala
require(SomePerm) {
  // multi-line code that creates view
}
```

If the `view` and `fallback` would be in separate args list, one could write:
```scala
requireWithFallback(SomePerm) {
  // multi-line code that creates view
}(fallback)
```
or
```scala
requireWithFallback(SomePerm) {
  // multi-line code that creates view
} {
  // multi-line fallback creation
}
```

Of course this is more cosmetic than practical, I'd put it to a vote ;)

Reply:
Side note - current signature is way better if views are mostly passed to the function as variable references, not entire blocks of code. That's why I mentioned most common usage patterns.

Reply:
From what I've seen I believe that the most common usage is this one:
```
require(Permission)(View)
```
The only time I've seen one with block of code was in AuthViewTest. @ddworak what do you think?

Reply:
I have to agree with @mkej, the possibility to use a block here is quite useful (and it's a actually a part of the examples on https://guide.udash.io/ext/authorization, which should be updated within the scope of this PR :smile: )

Reply:
Understood, I'll adjust the interface and update examples